python/interpret-core/setup.py

```diff
@@ -240,7 +240,7 @@ def run(self):
         "numpy>=1.11.1",
         "scipy>=0.18.1",
         "pandas>=0.19.2",
-        "scikit-learn>=0.18.1",
+        "scikit-learn>=0.24",
```

Comment:
Can scikit-learn 0.22 work here, or does 0.24 make the check_estimator significantly nicer?

Reply:
To be honest, I am not sure... The [documentation](https://scikit-learn.org/stable/modules/generated/sklearn.utils.estimator_checks.parametrize_with_checks.html) states `new in 0.24`. But it is not quite clear to me what changed.
Personally, I don't really like supporting `0.` release after the first stable release, but if you consider it important, I can test versions `0.22` till `0.24` and check if there are some issues.